.pre-commit-config.yaml

```diff
@@ -1,4 +1,9 @@
 exclude: ^misc/bench.py$
+
+default_language_version:
+  rust: 1.65.0
+  node: 18.13.0
```

Comment:
I couldn't compile to `node` with a old version, and when I updated `node` then my `pre-commit` doesn't work :( because it uses my `nodejs` (`node` for the system)

Reply:
Also it doesn't fail if you haven't the version pointed

Reply:
👍